src_assets/common/assets/web/styles/main.css

```diff
@@ -0,0 +1,35 @@
+.nav-link.active {
+  font-weight: 500;
+}
+
+
+
+
```

Comment:
Can you make the blank lines between a consistent number? I'm not a css expert so don't know what the standard is. Probably 1 blank line?

Reply:
Can do